.eslintrc.js

```diff
@@ -37,5 +37,5 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
   },
-  ignorePatterns: ['gatsby-*.js', 'jest', '*.md'],
+  ignorePatterns: ['gatsby-*.js', 'jest', '*.md', 'tooling/**'],
```

Comment:
It's linting my js file with ts rules which is causes issues I can't fix so here is the workaround. Should this work or what's wrong with our setup ?

Reply:
I never dived into this problem, don't know yet.